### PR TITLE
Round-end statistics

### PR DESCRIPTION
**Pull request by Lohikar**
This PR adds a system to display round statistics at the end of the round, with the option of logging said statistics to the feedback database.

Tracked stats:
- Most deaths by ckey.
- Number of people cloned.
- Number of times humans have fallen into openturfs.
- Number of times humans have died from falling into openturfs.
- Number of times humans have been gibbed.

### CODE REVIEW

File: code/controllers/subsystems/statistics.dm [hunk 1]
```diff
@@ -113,9 +125,23 @@
 	feedback_add_details("radio_usage","PDA-[pda_msg_amt]")
 	feedback_add_details("radio_usage","RC-[rc_msg_amt]")
 
+	for (var/datum/statistic/S in simple_statistics)
+		if (S.key)
+			log_debug("SSfeedback: Writing [S] to database.")
```
Comment (skull132) [inline code/controllers/subsystems/statistics.dm]: Debug necessary?

File: code/datums/statistic.dm [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	var/name
+	var/key
+	var/write_to_database = FALSE
+	var/broadcast_at_roundend = TRUE
+
+/datum/statistic/proc/set_value()
+	WARNING("Statistic [type] does not have set_value implemented.")
+
+/datum/statistic/proc/get_value()
+	WARNING("Statistic [type] does not have get_valie implemented.")
```
Comment (Arrow768) [inline code/datums/statistic.dm]: spelling mistake at valie

Comment (Lohikar) [inline code/datums/statistic.dm]: whoops

File: code/modules/multiz/movement.dm [hunk 3]
```diff
@@ -371,8 +371,16 @@
 	else
 		playsound(src.loc, "sound/weapons/smash.ogg", 75, 1)
 
+	// Stats.
+	SSfeedback.IncrementSimpleStat("openturf_human_falls")
+	addtimer(CALLBACK(src, .proc/post_fall_death_check), 2 MINUTES, TIMER_UNIQUE | TIMER_OVERRIDE)
+
 	return TRUE
 
+/mob/living/carbon/human/proc/post_fall_death_check()
```
Comment (Arrow768) [inline code/modules/multiz/movement.dm]: yes, Yes, YES we need that

**Pull request merged**